### ISSUE TITLE
remove ticker from top of homepage

### ISSUE THREAD

**Issue by akernest** [cassiopaia/labs.bitshares.org#21]
create bit asset coin icons and put in place of “how it works”, “Deposit”, and “Get Started” buttons, and place ticker info (price - relative to BTS | % change (24h) | volume (24h) directly under bit asset coin icons

**Comment by cassiopaia** [comment]
so deleting current feature button nav ? 

How it works - deposit - get started!?

**Comment by cassiopaia** [comment]
@svk31: can you pls help me on ticker update!?

**Comment by akernest** [comment]
this should be done after #19 because, at that point, How it works - deposit - get started buttons will be visible in the uniform header and no longer needed below.

**Comment by cassiopaia** [comment]
ok IMO it would be much more helpful to produce more content instead of destroying  initial design... just said!

**Comment by xeroc** [comment]
fully agree with @cassiopaia here .. let's pimp the content first .. 
content > design .. even if marketers do not agree
